lib/KCard/index.vue

```diff
@@ -34,7 +34,7 @@
           as progressive loading experience.
         -->
         <span
-          v-if="$slots.thumbnailPlaceholder"
+          v-if="$slots.thumbnailPlaceholder && !thumbnailSrc"
```

Comment:
One of the functions of the placeholder background and element (such as icon) is to show while the image is loading over it. I've tried to explain with the comment above "Show it even when thumbnail source is provided - then the placeholder as progressive loading experience." Adding this condition will break it. I assume there was a situation where it caused some unexpected problems - could you elaborate? Perhaps we can think of a solution while still keeping progressive loading experience feature.